ark/utils/plot_utils.py

```diff
@@ -105,5 +109,5 @@ def plot_overlay(predicted_contour, plotting_tif, alternate_contour=None, path=N
         # save as TIF if path supplied, otherwise display on screen
         if path is not None:
             io.imsave(path, rescaled)
-        else:
+        if show:
```

Comment:
Why did you change this from `else show` to `if show`? This is going to show the image no matter if `path` is specified or not. We only want to show the image if `path` is not specified.